.github/workflows/release-production.yml

```
@@ -20,6 +31,13 @@ jobs:
       - name: Get tag
         run: echo "TAG=${GITHUB_REF#refs/*/v}" >> $GITHUB_ENV
 
+      - name: Set default tag
+        run: echo "IMAGE_TAG=${{ env.REGISTRY }}/${{ github.repository }}:${{ env.TAG }}" >> $GITHUB_ENV
+
+      - name: Override tag if custom tag is provided
+        if: github.event.inputs.image_tag != ''
+        run: echo "IMAGE_TAG=${{ github.event.inputs.image_tag }}" >> $GITHUB_ENV        
```

Easier if you use the same env variable
```suggestion
        run: echo "TAG=${{ github.event.inputs.image_tag }}" >> $GITHUB_ENV        
```

---

Updated.